src/scripts/main.js

```diff
@@ -22,7 +22,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ulElement = document.createElement('ul');
+
+  for (const key in data) {
+    const liElement = document.createElement('li');
+
+    liElement.innerText = key;
+
+    if (Object.keys(data[key]).length > 0) {
```

Comment:
Use `Object.keys(data[key]).length` within the if statement is inefficient, as it's computed twice. Consider storing it in a variable before using it in the condition and inside the loop.